### PR TITLE
add e2e check for extensionmethod

### PR DESCRIPTION
**Pull request by ansyral**
@vwxyzh  @hellosnow @qinezh @chenkennt

### CODE REVIEW

**Comment by hellosnow** [comment]
:shipit:

**Comment by vwxyzh** [comment]
👍

File: src/docfx.website.themes/default/partials/class.tmpl.partial [hunk 1]
```diff
@@ -294,7 +294,7 @@
 {{/children}}
 {{/children}}
 {{#extensionMethods.0}}
-<h3 id="{{id}}">{{__global.extensionMethods}}</h3>
+<h3 class="extensionmethods">{{__global.extensionMethods}}</h3>
```
Comment (superyyrrzz) [inline src/docfx.website.themes/default/partials/class.tmpl.partial]: id="extensionmethods"

File: test/docfx.E2E.Tests/DocfxSeedSiteTest.cs [hunk 2]
```diff
@@ -187,6 +187,20 @@ public void TestReferencePage()
...
                 Assert.Contains(titleBase, results[0].Text);
             }
+
+            // check extension methods
+            results = _driver.FindElements(By.XPath("//h3[@class='extensionmethods']"));
+            if (results.Count > 0)
+            {
+                Assert.Equal(1, results.Count);
+                results = _driver.FindElements(By.XPath("//h3[@class='extensionmethods']/following-sibling::div/a"));
+                for (int i = 0; i < results.Count; i++)
```
Comment (superyyrrzz) [inline test/docfx.E2E.Tests/DocfxSeedSiteTest.cs]: Can test one result. Move to test the `Cat` page.

**Comment by ansyral** [comment]
@superyyrrzz  Updated

**Comment by superyyrrzz** [comment]
:+1:

**Pull request merged**